rust/feature-flags/src/router.rs

```diff
@@ -9,8 +9,8 @@ use crate::{
 
 #[derive(Clone)]
 pub struct State {
+    // TODO add writers when ready
     pub redis: Arc<dyn RedisClient + Send + Sync>,
-    // TODO: Add pgClient when ready
     pub postgres: Arc<dyn DatabaseClient + Send + Sync>,
```

Comment:
As above, your `State` object always has a DB connection, you can bubble that certainty down to all your other methods here